src/main/java/seedu/address/logic/commands/task/SearchTaskCommand.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.commands.task;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.task.TaskCliSyntax.PREFIX_DUE_DATE;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.room.Room;
+import seedu.address.model.task.DateTimeDue;
+import seedu.address.model.task.Task;
+
+/**
+ * Search a {@code Task} before a {@code DateTimeDue}.
+ */
+public class SearchTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "searchtask";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Searches a task in Covigent. "
```

Comment:
Searches task(s)* -> I think the output may be more than 1 task right?

Reply:
yup